docs/how-to/build-policies/policy-basics.mdx

```diff
@@ -0,0 +1,242 @@
+---
+sidebar_position: 1
+title: Policy Basics
+---
+
+import createRole from "/static/ui-videos/policy-basics/creating-a-role.mp4";
+import createResource from "/static/ui-videos/policy-basics/creating-a-resource.mp4";
+import createUser from "/static/ui-videos/policy-basics/creating-a-user.mp4";
+import assignMultipleRoles from "/static/ui-videos/policy-basics/assigning-multiple-roles.mp4";
+import givePermissions from "/static/ui-videos/policy-basics/giving-permissions-to-role.mp4";
+import createTenant from "/static/ui-videos/policy-basics/create-a-new-tenant.mp4";
+import switchTenant from "/static/ui-videos/policy-basics/switching-between-tenants.mp4";
+import switchView from "/static/ui-videos/policy-basics/changing-policy-view.mp4";
+
+Writing and maintaining good authorization policies is hard.
+The **[Policy Editor](https://app.permit.io/policy-editor)** plays a centric part into what we offer at Permit. It was
+designed to take away the hassle of dealing with complex code, and offer a UI solution that allows both novice and
+experienced users to create and edit authorization policies with ease - with code or low-code interfaces.
+
+We combine the power of a powerful authorization engine with the ease of use of a simple web interface, making us a very desired
+product withing many companies.
+You can manage policies from the **[Policy Editor](https://app.permit.io/policy-editor)** page.
+
+## What is a Policy?
+
+A policy is a way of defining rules of how permissions are granted. Within each policy, we focus on a hierarchy view of
+how permissions are applied.
+
+:::info
+A very basic policy consists of **user roles**, **actions**, and **resources**.
+:::
+
+:::info
+A policy **resides** inside an **environment**.
+:::
+
+## Supported Authorization Models
+
+There are three types of policies that we currently support:
```

Comment:
I'd like us to minimize the difference between models - as people don't really care about them.
Instead let's talk about roles, relationships and conditions - which can all be mixed together and added at different stages